multisig/simulations/arbitrumone/vip-008/index.ts

```diff
@@ -0,0 +1,110 @@
+import { expect } from "chai";
+import { Contract } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { forking, pretendExecutingVip } from "../../../../src/vip-framework";
+import {
+  RewardsDistributorConfig,
+  checkRewardsDistributor,
+  checkRewardsDistributorPool,
+} from "../../../../src/vip-framework/checks/rewardsDistributor";
+import vip008, {
+  COMPTROLLER_CORE,
+  REWARD_DISTRIBUTOR_CORE_0,
+  VARB_CORE,
+  VUSDC_CORE,
+  VUSDT_CORE,
+  VWBTC_CORE,
+  VWETH_CORE,
+  XVS,
+  XVS_AMOUNT,
+  XVS_STORE,
+  XVS_VAULT_PROXY,
+  XVS_VAULT_REWARDS_SPEED,
+} from "../../../proposals/arbitrumone/vip-008";
+import XVS_VAULT_ABI from "./abi/XVSVaultProxy.json";
+import XVS_ABI from "./abi/xvs.json";
+
+forking(221153914, () => {
+  const provider = ethers.provider;
+  let xvsVault: Contract;
+  let xvs: Contract;
+
+  describe("Post-VIP behavior", async () => {
+    before(async () => {
```

Comment:
I think we'll need to mint some XVS tokens and transfer them to the VTreasury on Arbitrum one in the simulation. Otherwise, the balance checks will fail.

We made something similar [here](https://github.com/VenusProtocol/vips/pull/301/files#diff-505051b856023361051740f6ea4931bf5fb2c92ee6cad92f3e2ed69f39467fb0R127-R131)

Reply:
The Treasury will hold the XVS tokens once bridged, we can then update the block numbe and check it.

Reply:
The multisig TX will be included in the same VIP. We'll execute first the VIP commands, and after some minutes the multisig TX commands. I would not wait for the execution of the VIP commands to fully simulate the multisig TX

Reply:
Done https://github.com/VenusProtocol/vips/pull/274/commits/2945d2fb86a760a01af34ac92f6072f0d767a3bd